### DISCUSSION TITLE
auto-convert templates from React/Next to Vue.. feasible ?

### DISCUSSION THREAD

**Discussion by vesper8** [kiliman/tailwindui-crawler#70] [General]
I just found out that the new templates are React only and all components are written in JSX. I find this really annoying since all my projects are in Vue and thus integrating the templates means a lot of adaptation.

I'm sure you've had a look at the new templates code.. how difficult do you think it might be to transform the .jsx files to .vue SFCs and if not handle a fully working conversion.. at least save some time by handling the repetitive tasks?

**Comment by kiliman** [comment]
I'm not a Vue expert. Can you take a sample and show the before and after? We may be able to use the `transformers/createVue.js` function to do some of the heavy lifting.
